specification/validated-relationships-service-api.yaml

```diff
@@ -247,7 +246,10 @@ paths:
 
             | HTTP status | Error code             | Description                                                                                                                               |
             | ----------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-            | 400         | `BAD_REQUEST`          | Bad request. For details, see the `diagnostics`                                                                                           |
+            | 400         | `BAD_REQUEST`          | Bad request.                                                                                                                              |
+            | 400         | `NOT_SUPPORTED`        | The request is not currently supported.                                                                                                   |
```

Comment:
Can NOT_SUPPORTED be removed, I don't think the error code is used for this endpoint